Android/Shared/src/main/java/com/bentley/sample/shared/ActivityTimer.kt

```diff
@@ -56,7 +72,7 @@ class ActivityTimer(private val nameTitle: String = "ACTIVITY") {
      * @param logger: The [ITMLogger] to use to log the times
      * @param title: Title to be printed before the checkpoints table
      */
-    fun logTimes(logger: ITMLogger, title: String) {
+    fun logTimes(logger: ITMLogger, appContext: Context, title: String) {
```

Comment:
appContext parameter needs to be added to function docs.

Reply:
Fixed.